python/pip.bzl

```diff
@@ -102,6 +102,20 @@ py_binary(
 """,
 )
 
+def pip2_import(**kwargs):
+    """A wrapper around pip_import that uses the `python2` system command."""
+    # We don't advertise "Use this for PY2 programs" because it's not clear
+    # whether this alias is always available, and on most (but not all) systems
+    # `python` is Python 2 anyway.
+    pip_import(python_interpreter = "python2", **kwargs)
```

Comment:
This won't work on macOS I guess

Reply:
IIRC macOS has `python` and `python2.7` but not `python2`, right?

I was ambivalent about whether to add `pip2_import` at all, or just leave that use case to `pip_import` (with or without a custom `python_interpreter` attr). I guess it's not ideal to encourage people to switch Python 2 deps to use `pip2_import` and thereby break mac users by default.

I suppose we could make this macro smarter to detect what platform it's running on and choose a different default. Thoughts?

Reply:
Correct. Yea I think related to https://github.com/bazelbuild/bazel/pull/9402 it would be safe to default this to `python2.7` on macOS, but I agree that this isn't a huge issue since this still provides complete customizability for users

Reply:
I'm going to revert the addition of `pip2_import` for the purposes of this PR. Filed #258 to track the feature of `pip_import` reliably detecting a suitable Python interpreter.